app/javascript/dashboard/routes/dashboard/commands/commandbar.vue

```diff
@@ -6,11 +6,12 @@
     hideBreadcrumbs
     :placeholder="placeholder"
     @selected="onSelected"
+    @closed="onClosed"
   />
 </template>
 
 <script>
-import 'ninja-keys';
+import '@chatwoot/ninja-keys';
```

Comment:
Used the `ninja-keys` from Chatwoot repo, to use the `closed` event

Reply:
We don't need this now, right? We can switch to the original package

Reply:
Yeah, sure